tribe/settings.py

```diff
@@ -14,8 +16,8 @@
 SITE_NAME = 'tribe'
 
 # Settings files
-SETTINGS_ROOT = dirname(abspath(__file__))
-SETTINGS_INI_FILES = normpath(join(SETTINGS_ROOT, 'settings'))
+settings_root = dirname(abspath(__file__))
```

Comment:
The variables whose names are all capital letters are used globally by Django framework.